elliottlib/cli/__main__.py

```diff
@@ -138,6 +141,12 @@ def remove_bugs(runtime, advisory, default_advisory_type, id):
             raise ElliottFatalError("Error: Could not locate advisory {advs}".format(advs=advisory))
 
         try:
+            if auto:
+                bug_ids = advs.errate_bugs
+            else:
+                bug_ids = [bzapi.getbug(i) for i in cli_opts.id_convert(id)]
```

Comment:
it looks like your goal here is to get not bug ids, but actual bug records. so i would call them just `bugs`
```suggestion
                bugs = advs.errata_bugs
            else:
                bugs = [bzapi.getbug(i) for i in cli_opts.id_convert(id)]
```
... although the error i get when playing with this indicates to me that `advs.errata_bugs` is probably a list of IDs.

```
$ ./elliott --group openshift-4.9 remove-bugs -a 83078 --auto
[...]
  File "/workspaces/elliott/elliottlib/cli/__main__.py", line 149, in <listcomp>
    click.echo(" {}".format(", ".join([str(b.bug_id) for b in bug_ids])))
AttributeError: 'int' object has no attribute 'bug_id'
```

Also, it's not likely to be important here, but if there is a way to retrieve many bugs at once from the bzapi, rather than one at a time... that would be nice.